demos/twitter/src/main/java/com/datatorrent/demos/twitter/TwitterStatusMediaTypeExtractor.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2015 DataTorrent, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datatorrent.demos.twitter;
+
+import com.datatorrent.api.DefaultInputPort;
+import com.datatorrent.api.DefaultOutputPort;
+import com.datatorrent.common.util.BaseOperator;
+
+/**
+ * <p>
+ * TwitterStatusMediaTypeExtractor class.
+ * </p>
+ *
+ * @since 0.3.2
+ */
+public class TwitterStatusMediaTypeExtractor extends BaseOperator
```

Comment:
This Operator does nothing, just passes the data to the output port. Is it actually required?